kaiax/gov/headergov/impl/consensus.go

```diff
@@ -116,6 +116,7 @@ func (h *headerGovModule) VerifyGov(header *types.Header) error {
 }
 
 func (h *headerGovModule) checkConsistency(blockNum uint64, vote headergov.VoteData) error {
+	//nolint:exhaustive
```

Comment:
Here is a brief discussion about this new appearance.

Which do you think is better: using the nolint annotation or adding a default case without the nolint annotation?

Reply:
fyi, there's an option called `default-signifies-exhaustive`.
It's false by default, so in case of the latter, we need to set it as true.
I think both are fine.